### DISCUSSION TITLE
How can i create an MSI rather then exe and how to by pass installation steps and install .msi silently without asking user options?

### DISCUSSION THREAD

**Discussion by kcs-arpitdixit** [ActivityWatch/activitywatch#1073] [General]
How can i create an MSI rather then exe and how to by pass installation steps and install .msi silently without asking user options?

**Comment by 0xbrayo** [comment]
Why would you want to install activitywatch "silently"? What are you trying to achieve?

**Reply by kcs-arpitdixit** [reply]
i want to install it without asking where to install on client's system

**Comment by BelKed** [comment]
I wouldn't say this is an ActivityWatch specific question, so a quick search on the web may give you some options, such as here:
https://stackoverflow.com/questions/50879667/python-files-to-an-msi-windows-installer

**Reply by 0xbrayo** [reply]
I think he wants to install activitywatch without user interaction. I could be wrong though.

**Reply by kcs-arpitdixit** [reply]
yes you are right i dont want to give an option to user where he can choose the path where he/she wants to install this service

**Comment by PhilippDehner** [comment]
You can use Winget, if this is an option
